taxi/tests/test_admin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminSiteTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="123123"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_superuser(
+            username="driver",
+            password="11111111",
+            license_number="AAA17171"
+        )
```

Comment:
Creating a driver as a superuser might not be appropriate if the driver is not intended to have admin privileges. Consider using `create_user` instead of `create_superuser` for the driver to better reflect typical user roles.